lib/ui/forgot_password/forgot_password_screen.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:go_router/go_router.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/ui/widget/input_field_widget.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+import 'package:survey_flutter_ic/utils/custom_app_bar.dart';
+
+class ForgotPasswordScreen extends StatefulWidget {
+  const ForgotPasswordScreen({super.key});
+
+  @override
+  ForgotPasswordState createState() => ForgotPasswordState();
+}
+
+class ForgotPasswordState extends State<ForgotPasswordScreen> {
+  AppBar get _appBar => CustomAppBar.backButton(
+        context: context,
+        onPressed: () => context.go('/'),
+      );
+
+  Widget get _background => SizedBox(
+      width: MediaQuery.of(context).size.width,
+      height: MediaQuery.of(context).size.height,
+      child: Image(
+        image: AssetImage(Assets.images.bgLoginOverlay.path),
+        fit: BoxFit.cover,
+        alignment: Alignment.center,
+      ));
+
+  Widget get _nimbleLogo => Padding(
+      padding: const EdgeInsets.only(left: 104.0, right: 104.0),
```

Comment:
Or we can write this way: 

```suggestion
      padding: const EdgeInsets.symmetric(horizontal: 104.0),
```